js/collections/profile/SocialAccounts.js

```diff
@@ -1,16 +1,8 @@
-import { guid } from '../../utils';
```

Comment:
Could we actually revive this? Sorry, I didn't document it at all, but this is a common pattern I'm using for the Profile and Settings nested collections for models that don't have an ID. The reason is that we are working off of clones and after a save, we merge the cloned data back into the main models (`app.profile`, `app.settings`). If those models don't have IDs, then the "change" events think they are new. For example, let's say you had save out 3 social accounts and now you go back to settings and add a fourth. If those guys don't have id's, then the people listening on the add event of `app.profile.socialAccounts` will get 4 add events fired, one for each one, because without an ID, it can't tell it already had the 3.

(just a quirk of how we're working with a clone and then merging it back into the main model)